src/Components/CreateImageWizard/ValidatedTextInput.tsx

```diff
@@ -30,6 +34,62 @@ interface HookValidatedTextInputPropTypes extends TextInputProps {
   warning?: string;
 }
 
+interface HookValidatedTextAreaPropTypes extends TextAreaProps {
+  dataTestId?: string | undefined;
+  ariaLabel: string | undefined;
+  value: string;
+  placeholder?: string;
+  stepValidation: StepValidation;
+  fieldName: string;
+  isEmpty: boolean;
+}
+
+interface HookValidatedTextInputWithButtonPropTypes
+  extends HookValidatedTextInputPropTypes {
+  togglePasswordVisibility: () => void;
+  isPasswordVisible: boolean;
+  isEmpty: boolean;
+}
+
+const getValidationState = (
+  value: string,
+  stepValidation: StepValidation,
+  fieldName: string,
+  isPristine: boolean,
+  isEmpty?: boolean
+): 'default' | 'error' | 'success' => {
+  if (isEmpty) return 'default';
+  if (isPristine) return 'default';
+  if (stepValidation.errors[fieldName] === 'default') return 'default';
+  return stepValidation.errors[fieldName] ? 'error' : 'success';
+};
+
+const ErrorHelperText = ({
```

Comment:
To be consistent with the rest of the code base, please create a new type for the props instead of defining them inline.